src/main/java/core/basesyntax/db/FileReaderImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.db;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    public FileReaderImpl(String s) {
+    }
+
+    public List<String> read(String fileName) {
+        List<String> lines = new ArrayList<>();
+        try {
+            lines = Files.readAllLines(Path.of(fileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file",e);

```

Comment:
The RuntimeException message 'Can't read file' is not very informative. Include more details about the issue, for instance, the file name.

Reply:
ok, done